src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -285,10 +286,11 @@ data SqlDelete = SqlDelete
   , sqlDeleteWith   :: [(SQL, SQL, Materialized)]
   }
 
--- | This is not exported and is used as an implementation detail in
--- 'sqlWhereAll'.
-newtype SqlAll = SqlAll
-  { sqlAllWhere :: [SqlCondition]
+-- | Type representing a set of conditions that are joined by 'OR'.
```

Comment:
This doesn't look right. The Sqlable instance of this type joins conditions with AND.

Reply:
Okay, I did not even saw there's `Sqlable SqlAll` instance.

There was this old comment, mentioning `sqlWhereAll` which does not exist, so I probably assumed it meant `sqlWhereAny`, because it's used there in `sqlWhereAnyImpl`.
> -This is not exported and is used as an implementation detail in 'sqlWhereAll'.

I will review/rework this.